public/app/features/explore/LogsSamplePanel.tsx

```diff
@@ -58,7 +58,7 @@ export function LogsSamplePanel(props: Props) {
     }
 
     const onSplitOpen = () => {
-      splitOpen({ query: logSampleQueries[0], datasourceUid: datasourceInstance.uid });
+      splitOpen({ queries: logSampleQueries, datasourceUid: datasourceInstance.uid });
```

Comment:
I sneaked this in as this is the place where we want to pass more queries. 😇

Reply:
*old man yelling at clouds* 😂